### PR TITLE
update the README with dependency information

### PR DESCRIPTION
**Pull request by JoshMiers-FS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,13 @@
 
 [![CircleCI](https://circleci.com/gh/fullstorydev/fullstory-babel-plugin-react-native.svg?style=svg)](https://circleci.com/gh/fullstorydev/fullstory-babel-plugin-react-native)
 
-FullStory's React Native babel plugin performs compile-time transformations to allow FullStory to access certain React Native attributes.
+FullStory's React Native babel plugin performs compile-time transformations to add FullStory-specific attributes to all React Native controls. This plugin should only be used in conjunction with `@fullstory/react-native`.
```
Comment (patrick-fs) [inline README.md:6]: We've typically added a "This is a WIP" message to the README until the package is deployed on NPM.

Comment (JoshMiers-FS) [reply README.md:7]: done

File: README.md [hunk 2]
```diff
@@ -2,11 +2,14 @@
 
 [![CircleCI](https://circleci.com/gh/fullstorydev/fullstory-babel-plugin-react-native.svg?style=svg)](https://circleci.com/gh/fullstorydev/fullstory-babel-plugin-react-native)
 
-FullStory's React Native babel plugin performs compile-time transformations to allow FullStory to access certain React Native attributes.
+Note: This is a WIP.
```
Comment (patrick-fs) [inline README.md]:
```suggestion
## Note: This is a Work in Progress and has yet to be published to NPM.
```

Comment (JoshMiers-FS) [reply README.md]: done

**Pull request merged**